scripts/release fails on windows because electron-installer-{debian,redhat} are unconditionally required

I was testing the release workflow for my work on #634 and discovered the windows release job fails with:

```
Error: Cannot find module 'electron-installer-debian'
Require stack:
- D:\a\brim\brim\scripts\release\install.js
- D:\a\brim\brim\scripts\release\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (D:\a\brim\brim\scripts\release\install.js:5:25)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (D:\a\brim\brim\scripts\release\index.js:6:17)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\a\\brim\\brim\\scripts\\release\\install.js',
    'D:\\a\\brim\\brim\\scripts\\release\\index.js'
  ]
}
```
https://github.com/brimsec/brim/runs/637254118?check_suite_focus=true

This happens because the electron installers for linux are in `optionalDependencies` (and unavailable on windows) but the requires of them in `scripts/release/install.js` are unconditional:

https://github.com/brimsec/brim/blob/d0db49d7dc65b8ad12b946dcc4997029eb164acd/scripts/release/install.js#L5

--- a/scripts/release/install.js
+++ b/scripts/release/install.js
@@ -79,6 +79,9 @@ module.exports = {
 
   debian: function() {
     console.log("Building deb package installer")
+    // https://github.com/brimsec/brim/issues/724
+    // electron-installer-debian isn't available on Windows. It's an
+    // optionalDependency, so the require can't be module-scoped.
     const installerDebian = require("electron-installer-debian")
     return installerDebian({
       ...defaultLinuxOpts,
@@ -89,6 +92,9 @@ module.exports = {
 
   redhat: function() {
     console.log("Building rpm package installer")
+    // https://github.com/brimsec/brim/issues/724
+    // electron-installer-redhat isn't available on Windows. It's an
+    // optionalDependency, so the require can't be module-scoped.
     const installerRedhat = require("electron-installer-redhat")
     return installerRedhat({
       ...defaultLinuxOpts,

--- a/scripts/release/install.js
+++ b/scripts/release/install.js
@@ -2,8 +2,6 @@
 const fs = require("fs")
 const os = require("os")
 const installerWin = require("electron-winstaller")
-const installerDebian = require("electron-installer-debian")
-const installerRedhat = require("electron-installer-redhat")
 const createDMG = require("electron-installer-dmg")
 const createZip = require("electron-installer-zip")
 const path = require("path")
@@ -81,6 +79,7 @@ module.exports = {
 
   debian: function() {
     console.log("Building deb package installer")
+    const installerDebian = require("electron-installer-debian")
     return installerDebian({
       ...defaultLinuxOpts,
       ext: "deb",
@@ -90,6 +89,7 @@ module.exports = {
 
   redhat: function() {
     console.log("Building rpm package installer")
+    const installerRedhat = require("electron-installer-redhat")
     return installerRedhat({
       ...defaultLinuxOpts,
       ext: "rpm",

